components/common/SectionContent.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+interface SectionContentProps {
```

Comment:
@Yoonsoyoung02 
여기 type으로 수정해주세요!!
```suggestion
type Props = {
```

Reply:
헛 죄송합니다!  변경했습니다!!